docs/source/conf.py

```diff
@@ -200,7 +200,7 @@
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-html_logo = "../pymc_logo.jpg"
+html_logo = "../logo/PyMC.png"
```

Comment:
In the diff above the folder is called "logos"?

And I don't see it in the docs preview either